pipelines/jobs/configurations/jdk17_pipeline_config.groovy

```diff
@@ -106,10 +97,10 @@ class Config17 {
                 os                  : 'windows',
                 arch                : 'x86-32',
                 additionalNodeLabels: 'win2012&&vs2017',
-                buildArgs : [
+                buildArgs           : [
                         hotspot : '--jvm-variant client,server'
                 ],
-                test                : ['sanity.openjdk']
+                test                 : 'default'
```

Comment:
Despite a lot of white space changes in here, this one seems to have an extra space (assuming the goal was to line up all the `:` characters)

Reply:
Fixed